apps/src/applab/api.js

```diff
@@ -275,6 +275,19 @@ exports.setText = function (elementId, text) {
                            'text': text });
 };
 
+exports.getValue = function (elementId) {
```

Comment:
Hi Mike!  This is looking awesome so far.  I have a special request: Can we change these two blocks from `getValue`/`setValue` to `getNumber`/`setNumber`?

I know it's a little less traditional, but we've been talking internally about how to help students handle types (esp. with regard to our data API) and the concept of a `getNumber` parallel to our `getText` command came up.  We realized your PR would be a great place to introduce it.  No need to change the functionality for now, but we'll eventually extend this so you could call `getNumber` for other field types as a sort of get-and-cast operation.

Thanks!